src/epcpm/cli/main.py

```diff
@@ -102,7 +103,7 @@ def build(
         target_directory=target_path,
         paths=paths,
         first_time=False,
-        skip_sunspec=skip_sunspec,
+        skip_output=skip_sunspec,
```

Comment:
is this output supposed to drive different specs ? ... why name change

Reply:
I think Alex questioned this as well. Not sure if in this PR or on slack.

The name change in `importexport.py` is because the `skip_sunspec` name isn't quite accurate in the `importexport` module. It really means "skip SunSpec and static modbus output", but I shortened to `skip_output`. 

I could go further and remove all mentions of `skip_sunspec` in the code and switch them to `skip_output`. I could leave it. I could do something different. Any suggestions?

Reply:
no changes necessary.  I was thinking this variable drove more process steps.  I believe you made the right call